hack/test-go.sh

```diff
@@ -104,7 +109,7 @@ if [[ -n "${KUBE_COVER}" && -n "${OUTPUT_COVERAGE}" ]]; then
     mkdir -p "$OUTPUT_COVERAGE/$test_package"
     KUBE_COVER_PROFILE="-coverprofile=$OUTPUT_COVERAGE/$test_package/profile.out"
 
-    go test $KUBE_RACE $KUBE_TIMEOUT $KUBE_COVER "$KUBE_COVER_PROFILE" "$test_package" "${@:2}"
+    go test $KUBE_RACE $KUBE_TIMEOUT $KUBE_COVER "$KUBE_COVER_PROFILE" ${TESTFLAGS} "$test_package"
```

Comment:
do we ever have a test flag with a space in it? I *think* the old style would handle that, not sure this style does